build_tooling/build_many_linux_image.sh

```diff
@@ -44,7 +44,7 @@ RUN rpmkeys --import 'https://keyserver.ubuntu.com/pks/lookup?op=get&search=0x3F
             s/#?exclude.*/exclude=.edu/' /etc/yum/pluginconf.d/fastestmirror.conf
 ADD sccache /usr/local/bin/
 RUN yum update -y && \
-    yum install -y zip jq less devtoolset-11-gdb \
+    yum install -y zip jq less devtoolset-11-gdb perl-IPC-Cmd \
```

Comment:
Why do we need `perl-IPC-Cmd` ?

Reply:
It is required by v3:
https://github.com/man-group/ArcticDB/actions/runs/8649691810/job/23716646005
```
Can't locate IPC/Cmd.pm in @INC (@INC contains: /usr/local/lib64/perl5 /usr/local/share/perl5 /usr/lib64/perl5/vendor_perl /usr/share/perl5/vendor_perl /usr/lib64/perl5 /usr/share/perl5 .) at -e line 1.
  BEGIN failed--compilation aborted at -e line 1.
  CMake Error at buildtrees/versioning_/versions/openssl/53fb72a19981a3439cd569f58b722ca6cd26e23d/unix/portfile.cmake:27 (message):
    
  
    Perl cannot find IPC::Cmd.  Please install it through your system package
    manager.
```